README.md

```diff
@@ -117,3 +117,5 @@ To create ROOT files including the measured SFs please install [`TauIDSFs` tool]
 Modify the `TauIDSFs/utils/createSFFiles.py` script to include your measured SFs into the script. 
 Finally, run the `TauFW/scripts/tau_createROOT.sh` to generate your ROOT files. They will be created into `TauFW/scripts/data/`
 IMPORTANT: please comment and do not delete older SFs
+
+The latest code used to create root files for SF can be taken from [here](https://github.com/saswatinandan/TauFW/blob/master/Fitter/createroot_TES.py)
```

Comment:
Should this link be updated?

Reply:
I used this code to produce the root files containing all the sfs. If there is any other code to produce this, it can be linked here.

Reply:
I think that [this](https://github.com/cms-tau-pog/TauFW/tree/master/Fitter/TauES_ID) should be the README to enrich/update with the latest version of scripts running command.